config/settings.yml

```diff
@@ -1639,6 +1639,7 @@ vaos:
     client_assertion_type: urn:ietf:params:oauth:client-assertion-type:jwt-bearer
     client_id: fake_client_id
     grant_type: client_credentials
+    key: <%= ENV['vaos__eps__key'] %>
```

Comment:
This is a very generic name. Maybe change this to something more specific. token_key?